### PR TITLE
VIH-8535 Reduced bug rate of the staff member status not being correct

### PR DESCRIPTION
**Pull request by AWildBugAppeared**
This reduces the rate of the bug by fixing the property HostInAnotherHearing from returning true in cases when it should be false.

### CODE REVIEW

File: VideoWeb/VideoWeb/Mappings/ParticipantStatusResponseForVhoMapper.cs [hunk 1]
```diff
@@ -23,7 +24,8 @@ public class ParticipantStatusResponseForVhoMapper : IMapTo<Conference, IEnumera
                 .Select(x =>
                 {
                     var status = Enum.Parse<ParticipantStatus>(x.ParticipantStatus.ToString());
-                    var hostInHearing = hostsInHearings.Where(j => j.Username == x.Username && j.Id != x.Id);
+                    var hostInHearing = hostsInHearings.Where(j => j.Username == x.Username && j.Id != x.Id && 
+                        (j.Status == ParticipantState.InHearing || j.Status == ParticipantState.Available));
```
Comment (FarronF) [inline VideoWeb/VideoWeb/Mappings/ParticipantStatusResponseForVhoMapper.cs:28]: Should we check for InConsultation here as well?

Comment (AWildBugAppeared) [reply VideoWeb/VideoWeb/Mappings/ParticipantStatusResponseForVhoMapper.cs:28]: That makes sense. I will add that

**Review by ChrisPaulP — APPROVED** [review]

**Review by FarronF — APPROVED** [review]

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.html [hunk 2]
```diff
@@ -6,9 +6,9 @@
   <div class="header-bar">
     <div class="govuk-body" [class]="!isJohRoom && isPrivateConsultation ? 'header-content-grid-with-lock' : 'header-content-grid'">
       <ng-container *ngIf="!isJohRoom && isPrivateConsultation">
-          <div class="room-title">
-              <h1 #roomTitleLabel class="room-title-label">{{ getRoomName() }}</h1>
-          </div>
+        <div class="room-title">
```
Comment (omio-ma) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.html:9]: were there no changes for this file could we omit it if thats the case?

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-shared/participant-status-base/participant-status-base.component.ts [hunk 3]
```diff
@@ -101,7 +101,6 @@ export abstract class ParticipantStatusDirective {
         if (!this.participants) {
             return;
         }
```
Comment (omio-ma) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-shared/participant-status-base/participant-status-base.component.ts:103]: same here appears this change was not required?

**Review by omio-ma — APPROVED** [review]
leaving this pr approved, I dont see an issue leaving those whitespace changes in.

**Pull request merged**